recipe/meta.yaml

```diff
@@ -1,45 +1,43 @@
 {% set name = "keras-tuner" %}
-{% set version = "1.2.1" %}
+{% set version = "1.4.7" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 21478f73e6f1ce5a34e8b378eb10dc935a5e6e6eb74deb46131cd23653d13422
+  sha256: 6befd25ee81476e6207d8ca7ed7dc674b8194437cfa0b127294cd00da905ff22
 
 build:
-  noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<37]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python
+    - setuptools
+    - wheel
   run:
-    - python >=3.7
-    - Ipython
-    - kt-legacy
-    - numpy
+    - python
+    - keras
     - packaging
     - requests
-    - scikit-learn
-    - scipy
-    - tensorboard
-    - tensorflow >=2.0
-
+    - kt-legacy
+    # extras
```

Comment:
extras_require should go in run_constrained, but only if there's a version constraint against them upstream.